### DISCUSSION TITLE
conf mail problem

### DISCUSSION THREAD

**Discussion by nassim-01** [community/community#118287] [Mobile]
### Select Topic Area

General

### Body

hello,
I'm having a problem with conf mail I tried to change margin but the code cracked or something I tried to change the code from github files but it didn't work the mail is a mess please can you help?
this is how the mail is showing 
![Screen Shot 2024-04-05 at 8 02 35 PM](https://github.com/community/community/assets/140714571/b5caca4b-c295-4107-bcbe-7be57b90a3ab)

**Comment by Aiakos13** [comment]
response #2

**Reply by nassim-01** [reply]
thank you for your response I hope this is what you mean I changed category to mobile hopefully I can find help
